### DISCUSSION TITLE
DMR-6X2 Pro Support

### DISCUSSION THREAD

**Discussion by thom1879** [hmatuschek/qdmr#335] [General]
Are there any plans for support for the BTECH DMR-6X2 Pro Model?

**Comment by hmatuschek** [comment]
If the DMR-6X2 Pro relates to the DMR-6X2 like the Pro versions of the D878UV relate to the originals, the DMR-6X2 Pro should work out of the box. If the device, however, is not detected, I need to investigate that. Then, the Pro device identifies itself differently to the CPS and there might be some differences in the codeplug structure.

**Comment by allesand** [comment]
Is that the same model as in https://github.com/hmatuschek/qdmr/issues/260 ?

**Reply by thom1879** [reply]
Yes, thanks, I didn’t see that.

**Reply by allesand** [reply]
For the time until qdmr supports the DMR-6X2 Pro: I remember to have read that you can flash the corresponding Anytone firmware on the device - http://anytonetechzone.byethost7.com/878techmods.htm has some hints. I do not know whether that works without exchanging the boot loader (which means opening the device), which was needed when "upgrading" from an 868 or DMR-6X2 to an 878 (https://www.f5uii.net/en/convert-at-d868uv-to-at-d878uv-analog-aprs-roaming/)
